src/test/java/seemedics/service/triage/engine/LocalDirectoryTriageProtocolsTest.java

```diff
@@ -18,10 +18,17 @@
 public class LocalDirectoryTriageProtocolsTest {
     private LocalFilesTriageProtocols localFilesTriageProtocols = new LocalFilesTriageProtocols();
 
+    private class UnitTestTriageProtocolsSource implements TriageProtocolsSource{
+
+        @Override
+        public Path get_pathToProtocols() {
+            return Paths.get("src/test/resources/seemedics/model/triage/protocolsDirectory");
```

Comment:
For easy refactoring in the future it is better to fully separate resources between tests (especially between unit vs integration tests) - in this case all tests use the same protocols which will lead to complications in future refactoring.